CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 #### 3.1.11 (To be released)
 
+  Improvements
+     * NO_PROXY setting now supports IP ranges so you can specify whole subnet to be excluded from proxy traffic eg. 192.168.0.1/8
+   
   Bugs
-   * Impersonation parameters not set in withRequestConfig - https://github.com/fabric8io/kubernetes-client/pull/1037
+     * Impersonation parameters not set in withRequestConfig - https://github.com/fabric8io/kubernetes-client/pull/1037
 
+>>>>>>> upstream/master
```

Comment:
Please fix the conflict and squash in one commit.

Reply:
I missed that one sorry, will fix and squash.

Reply:
Thanks man :-)

Reply:
So I managed to squash all my commits into one. I updated my fork from original repo and now PR includes commits from there - not sure if its ok.

Reply:
It doesn't seem to be ok.